### ISSUE TITLE
Detect the permitted attributes of the input element by case-sensitive.

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#262]
```sh
echo '<input type="checkBox" checked>' | npx markuplint
```

```
<markuplint> error: The "checked" attribute is not allowed (invalid-attr) /path/to/<AnonymousFile>:1:24
   1: <input•type="checkBox"•checked>
                                ^^^^^^^ 
   2:
```
